src/webhint-theme/layout/partials/home.ejs

```diff
@@ -18,18 +18,17 @@
                     <div class="module feature">
                         <img src="/images/webhint_puzzle.svg" alt="" role="presentation" />
                         <h3 class="subtitle subtitle--home">Develop</h3>
-                        <p class="feature__highlight">Use the <a href="https://marketplace.visualstudio.com/items?itemName=webhint.vscode-webhint">VS Code extension</a> to get real-time feedback in your IDE when writing code</p>
+                        <p class="feature__highlight">Use the <a href="/docs/user-guide/extensions/vscode-webhint/">VS Code extension</a> to get real-time feedback in your IDE when writing code.</p>
                     </div>
                     <div class="module feature">
                         <img src="/images/webhint_firefox.svg" alt="" role="presentation" />
                         <h3 class="subtitle subtitle--home">Debug</h3>
-                        <p class="feature__highlight">The <a href="https://webhint.io/docs/user-guide/extensions/extension-browser/">browser extension</a> lets you scan a site from within your browser's DevTools.</p>
-
+                        <p class="feature__highlight">Test your site from the browser with the <a href="/docs/user-guide/extensions/extension-browser/">DevTools extension</a>. Using Edge? webhint now ships with Edge DevTools!</p>
```

Comment:
Should we link to the Edge Issues panel documentation here as well for users to learn more?
https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/issues/

Reply:
I'll leave this for once we integrate the content updates from the hint repo.